stl/modules/std.compat.ixx

```diff
@@ -0,0 +1,532 @@
+export using std::fma;
+export using std::fmax;
+export using std::fmin;
+export using std::round;
+export using std::trunc;
+export using std::remainder;
+export using std::remquo;
+export using std::copysign;
+export using std::nan;
+export using std::nextafter;
+export using std::scalbn;
+export using std::scalbln;
+export using std::nexttoward;
+export using std::tgamma;
+export using std::acoshf;
+export using std::asinhf;
+export using std::atanhf;
+export using std::cbrtf;
+export using std::erff;
+export using std::erfcf;
+export using std::expm1f;
+export using std::exp2f;
+export using std::hypotf;
+export using std::ilogbf;
+export using std::lgammaf;
+export using std::log1pf;
+export using std::log2f;
+export using std::logbf;
+export using std::llrintf;
+export using std::lrintf;
+export using std::nearbyintf;
+export using std::rintf;
+export using std::llroundf;
+export using std::lroundf;
+export using std::fdimf;
+export using std::fmaf;
+export using std::fmaxf;
+export using std::fminf;
+export using std::roundf;
+export using std::truncf;
+export using std::remainderf;
+export using std::remquof;
+export using std::copysignf;
+export using std::nanf;
+export using std::nextafterf;
+export using std::scalbnf;
+export using std::scalblnf;
+export using std::nexttowardf;
+export using std::tgammaf;
+export using std::acoshl;
+export using std::asinhl;
+export using std::atanhl;
+export using std::cbrtl;
+export using std::erfl;
+export using std::erfcl;
+export using std::expm1l;
+export using std::exp2l;
+export using std::hypotl;
+export using std::ilogbl;
+export using std::lgammal;
+export using std::log1pl;
+export using std::log2l;
+export using std::logbl;
+export using std::llrintl;
+export using std::lrintl;
+export using std::nearbyintl;
+export using std::rintl;
+export using std::llroundl;
+export using std::lroundl;
+export using std::fdiml;
+export using std::fmal;
+export using std::fmaxl;
+export using std::fminl;
+export using std::roundl;
+export using std::truncl;
+export using std::remainderl;
+export using std::remquol;
+export using std::copysignl;
+export using std::nanl;
+export using std::nextafterl;
+export using std::scalbnl;
+export using std::scalblnl;
+export using std::nexttowardl;
+export using std::tgammal;
+export using std::fpclassify;
+export using std::signbit;
+export using std::isfinite;
+export using std::isinf;
+export using std::isnan;
+export using std::isnormal;
+export using std::isgreater;
+export using std::isgreaterequal;
+export using std::isless;
+export using std::islessequal;
+export using std::islessgreater;
+export using std::isunordered;
+
+// <csetjmp>
+export using std::jmp_buf;
+export using std::longjmp;
+
+// <csignal>
+export using std::sig_atomic_t;
+export using std::raise;
+export using std::signal;
+
+// <cstdarg>
+export using std::va_list;
+
+// <cstddef>
+export using std::ptrdiff_t;
+export using std::size_t;
+export using std::max_align_t;
+

```

Comment:
I think `nullptr_t` should be exported to the global namespace. Although LWG-3484 has been filed now, I believe we should keep the current design, as `nullptr_t` has become a part of the C standard library since C23 (see also [WG14-N3042](https://open-std.org/JTC1/SC22/WG14/www/docs/n3042.htm), [WG14-N3048](https://open-std.org/JTC1/SC22/WG14/www/docs/n3048.htm)).

Reply:
I'll export `nullptr_t` in the global namespace (this is slightly forward-looking, but it ultimately aligns with the intended design). After talking with Casey on Discord, I've filed #3111 for followup - the intended design was not to export `lerp` or `byte` in the global namespace, as classic headers never provided that.